.travis.yml

```diff
@@ -23,6 +26,10 @@ env:
     - CONTAINER_RUNTIME=docker
     - JOBS=4
 
+os:
+- linux
+- linux-ppc64le
```

Comment:
Can we change the order of the jobs to first run the lints and then the tests like before?